idaes_examples/notebooks/docs/surrogates/alamo/alamo_flowsheet_optimization_src.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": null,
```

Comment:
This notebook looks like it was originally committed withe with all cells executed but these changes appear to have been done with the cells cleared. Was this a deliberate choice?

Reply:
The cells were cleared on my local copy when I opened it, but I thought it was generally advised to clear cells before pushing. Please let me know if it would be better to execute the cells first.

Reply:
@dangunter might need to weight in with how these get processed and displayed in the docs, but I think in the past this has depended on the purpose of the notebook. For those that are intended to be static examples (as opposed to tutorials) I think we left the executed cells in the notebooks so that users could directly see the results (although the infrastructure might run the cells for us anyway).